### DISCUSSION TITLE
Need English support?

### DISCUSSION THREAD

**Discussion by Hanxven** [LeagueAkari/LeagueAkari#97] [Ideas]
Most of the users come from China. If there's anyone needs English support pls let me know.

**Comment by YingLingWei** [comment]
Hey i really like the app and the ui that is used here but i dont understand chinese im using my phone to translate it for me at the moment with google lense
if it would be possible to add english support that would be awsome

**Reply by Hanxven** [reply]
Ok. Will implement it in 1.2.6 or later.

**Reply by YingLingWei** [reply]
ty ♥ 
my phone gallery is only photos of the options for googlr lense XD

**Comment by YingLingWei** [comment]
Hey is there already a plan for adding english support 🤔🤔

**Reply by Hanxven** [reply]
The software still needs some refactoring for now. English support may be delayed a lot

**Reply by Hanxven** [reply]
The current structure is highly coupled, so I don't know how to handle the addition of another language suuport :(

**Reply by YingLingWei** [reply]
:( i turst in you 😭 
i really love how the app looks and behaves so having it in a language that i understand without google translate would be really sick

**Comment by YingLingWei** [comment]
is there news on english support ?

**Reply by Hanxven** [reply]
Still developing on branch 'refactor'

**Reply by Hanxven** [reply]
Alternatively, could you assist me in translating some English? There might be certain gaming-specific terms that I may struggle to translate accurately.

**Reply by YingLingWei** [reply]
i mean the translation i did for myself is just using my google pixel combined with google lense so that translation is not that perfect aswell

**Comment by Hanxven** [comment]
The English support is done and will be released in the next version (v1.3.0).

[\[GitHub Action\]: League Akari-1.3.0-rabi.2+a9a0c46](https://github.com/Hanxven/LeagueAkari/actions/runs/11908461590/artifacts/2205873359)

**Reply by YingLingWei** [reply]
Hey just as feedback i really love how you did the ui
![image](https://github.com/user-attachments/assets/0b7160a5-909b-47ef-84ef-47a1979e5fc2)
its incredible ♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥♥
its AWSOME

**Reply by Hanxven** [reply]
Currently this version has Bugs now, will be fixed in tomorrow's release v1.3.1.

**Reply by YingLingWei** [reply]
i have a question how is this supposed to work 
![image](https://github.com/user-attachments/assets/01f75bff-b0fe-4527-98c9-32fd2dc4c71c)

will it create an overlay or where will the timers show

**Reply by Hanxven** [reply]
It doesn't support overlay for now, what it do is just display a timer at sidebar.

**Reply by Hanxven** [reply]
@YingLingWei Hi, could you help me test the availability of SGP in EUW region? I'll send you a special version to check if it can work properly.

**Reply by YingLingWei** [reply]
> @YingLingWei Hi, could you help me test the availability of SGP in EUW region? I'll send you a special version to check if it can work properly.

Hey just saw the massage 
uhh i could try if you tell me what to do

**Reply by Hanxven** [reply]
Can you take a screenshot of this page? (after conencted to the client)

![image](https://github.com/user-attachments/assets/55f1ee02-b939-42a6-87b0-d7c1b0f92851)

**Reply by YingLingWei** [reply]
i will try yes

**Reply by YingLingWei** [reply]
i censored the auth key since i assume that could be used for smtg bad? maybe 
![Screenshot_1](https://github.com/user-attachments/assets/ef866c6b-0e65-47b0-a184-094ba50343cc)

**Reply by Hanxven** [reply]
Try the new TEST version:

[LA_133_ylw_1](https://drive.google.com/file/d/1V4Yi0h_5fBKv_LPTiq8uSDxZRH7Pyaww/view?usp=sharing)

This version added sgp server support for EUW region. I hope it works.

Make sure these settings are enabled:

![image](https://github.com/user-attachments/assets/1c92b387-15a8-4966-9cd6-3578c5345308)

![image](https://github.com/user-attachments/assets/6c3d7def-6223-4df2-bafc-0480a21ea5ed)

---

The expected additional functions should be:

- Faster match history loading
- Match history can be filtered by queue type (e.g. ARAM, Ranked, URF, and etc.)
- In-game status specatable. (If a player is already in game, you can see it in the side of his tab, and launch spectate)
- Avoided league client crashing in champ-select phase

> Due to Vanguard, game spectate called by commandline is not allowed. Only LCU API can be used.

> forgive my poor English

**Reply by YingLingWei** [reply]
will test with the version you linked 
where is the respawntimer displayed now ? in the app itself ? or were do i need to look

**Reply by YingLingWei** [reply]
appears to be working match history loads fast but normal matches dont seem to get filtered ranked aram seems to be working 
![image](https://github.com/user-attachments/assets/a2f05e06-5bb8-4049-aa48-bf19db8dd4ef)
![image](https://github.com/user-attachments/assets/25fb9b2c-243d-470a-bd03-f31f6771cd81)

**Reply by Hanxven** [reply]
## Respawn Timer

Like this, displaying by the sidebar. If it doesn't appear, maybe a bug happened :(

![image](https://github.com/user-attachments/assets/4f57ec1e-f2bb-4e06-b764-3e1d132d81fe)

## Normal Mode

It refers to the legacy Summoner Rift 5x5 mode, maybe it's replaced by Quickplay in riot servers (but still available in TENCENT servers now)

## Other

This section shows the ongoing game, if everyghing's ok

![image](https://github.com/user-attachments/assets/7b466be6-2633-4b1c-af8e-23772da024d3)

**Reply by YingLingWei** [reply]
> ## Respawn Timer
> 
> Like this, displaying by the sidebar. If it doesn't appear, maybe a bug happened :(
> 
> ![image](https://private-user-images.githubusercontent.com/52438792/394379182-4f57ec1e-f2bb-4e06-b764-3e1d132d81fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4NTIyMTIsIm5iZiI6MTczMzg1MTkxMiwicGF0aCI6Ii81MjQzODc5Mi8zOTQzNzkxODItNGY1N2VjMWUtZjJiYi00ZTA2LWI3NjQtM2UxZDEzMmQ4MWZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEwVDE3MzE1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA1M2E0MWNiNmY1N2VkNGU1MjgyZWYyNzRhNmVmM2NhOGRkNTE3MTFiYThiZGRiNzlmNTFmNTFjOTVjNTc1YmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.68RtG0BeNxGckdED4e3ulCicGCYX0zSg_7OHZZVLip0)
> ## Normal Mode
> 
> It refers to the legacy Summoner Rift 5x5 mode, maybe it's replaced by Quickplay in riot servers (but still available in TENCENT servers now)
> ## Other
> 
> This section shows the ongoing game, if everyghing's ok
> 
> ![image](https://private-user-images.githubusercontent.com/52438792/394380631-7b466be6-2633-4b1c-af8e-23772da024d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4NTIyMTIsIm5iZiI6MTczMzg1MTkxMiwicGF0aCI6Ii81MjQzODc5Mi8zOTQzODA2MzEtN2I0NjZiZTYtMjYzMy00YjFjLWFmOGUtMjM3NzJkYTAyNGQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEwVDE3MzE1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxMjNmNGM3NzZhY2Y4YmJkZGI5YjU3NjdiYzIwM2E5ZmM0NDRiZTU3ZmFhMjcwNTQ5NTI3YzE2ZGQzNWEzN2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UOeURnJvN6SdX3PcZZQ_c_uFyJxV1cenVzCWOBggsy0)

where do i find this ? 
![image](https://github.com/user-attachments/assets/81dcf8e7-d8d4-4620-804b-9e2893577fe0)
i only have this in the normal sidebar

**Comment by YingLingWei** [comment]
and your english is good no need to excuse yourself for it 

![1229266660628627517](https://github.com/user-attachments/assets/28992acd-7900-4e20-888d-5ac51e02f446)

**Comment by YingLingWei** [comment]
![image](https://github.com/user-attachments/assets/59c58bff-9d4b-4fe8-a375-7cde75fc9280)
normal should be draft pick right ?

**Comment by Hanxven** [comment]
Respawn timer only shown when death.

Normal Mode here is ~Draft Pick~ Blind Pick. I hardly play any modes other than ARAM, so im not sure about it. The text 'normal' comes from league client game data.

![image](https://github.com/user-attachments/assets/be5bb3d0-7aaa-4438-a20a-ac901e3862c0)

**Reply by YingLingWei** [reply]
ohhh ok so wich sorting would i need to use for draft pick ? or is that not in yet

**Reply by Hanxven** [reply]
No more things. I have to go bed since it's too late. Let's continue tomorrow
